### PR TITLE
Fix: Add missing counts to Director review widget and correct title casing - 992

### LINKED ISSUE #992: LCFS - IDIR user with a director role dashboard widget counter
**Describe the Bug**
IDIR users with a director role are currently unable to see the count of recommended transfers, Initiative Agreement (IA) agreements, Admin Adjustments, and recommended Compliance Reports from managers on the "Director Review" widget on the dashboard. 

This widget is intended to provide an overview of transfers, reports, and transactions that require director-level approval or recording, but it is not functioning as intended. 

Additionally, the widget title "Director Review" should use a lowercase "r" in "review" to align with design standards.

**Expected Behaviour**
- The Director Review widget should display the count of:
- Recommended Transfers
- IA Agreements
- Admin Adjustments
- Recommended Compliance Reports from managers
- The title of the widget should be "Director review" (with a lowercase "r").

**Actual Behaviour**
- The widget is not showing any counts for the relevant items, preventing directors from easily accessing information on what requires their attention.
- The title currently reads "Director Review" with an uppercase "R" in "review," which does not conform to design standards.

**Implications**
- Directors are unable to see at-a-glance counts of the transfers, reports, and transactions that require their review and approval, reducing the efficiency of the dashboard.
- The inconsistency in the title formatting does not adhere to the design standards, affecting visual consistency.

**Steps To Reproduce**
Steps to reproduce the behaviour:
User/Role: Log in as an IDIR user with a Director role.

1. Navigate to the Dashboard.
2. Look at the Director Review widget and observe that the counts for Recommended Transfers, IA Agreements, Admin Adjustments, and Recommended Compliance Reports are not displayed.
3. Notice that the widget title reads "Director Review" instead of "Director review".

### PR DESCRIPTION
**Pull request by hamed-valiollahi**
This PR fixes the Director review widget by adding missing counts. It also updates the widget title to use lowercase 'r' for consistency with design standards.

Closes #992

### CODE REVIEW

File: backend/lcfs/web/api/compliance_report/repo.py [hunk 1]
```diff
@@ -67,7 +67,16 @@ def apply_filters(self, pagination, conditions):
...
             if filter.field == "status":
                 field = get_field_for_filter(ComplianceReportStatus, "status")
-                filter_value = getattr(ComplianceReportStatusEnum, filter_value.title())
+                # Normalize the status value by making it lowercase and replacing spaces with underscores
+                normalized_value = filter_value.replace(" ", "_").lower()
+
+                # Match enum values without case-sensitivity
+                matching_enum_value = None
+                for enum_val in ComplianceReportStatusEnum:
+                    if enum_val.name.lower() == normalized_value:
```
Comment (dhaselhan) [inline backend/lcfs/web/api/compliance_report/repo.py:76]: code smell: It's odd to use enum_val.name, remember that this gets the name of the enum variable not the value of the enum. You should probably be using the enum itself most of the time, as changing the name of an enum should not break code.

Comment (hamed-valiollahi) [reply backend/lcfs/web/api/compliance_report/repo.py]: Thanks, Daniel, for pointing this out, I fixed the issue.

**Review by dhaselhan — APPROVED** [review]
Small conflicts then LGTM 👍

**Comment by hamed-valiollahi** [comment]
> Small conflicts then LGTM 👍

I fixed the conflict and will merge it now. Thanks for reviewing my PR!

**Pull request merged**